.pipelines/npm/npm-cni-integration-test.yaml

```diff
@@ -16,6 +16,7 @@ jobs:
     displayName: "NPM k8s E2E"
     dependsOn: ${{ parameters.dependsOn }}
     condition: and( and( not(canceled()), not(failed()) ), ${{ or(contains(parameters.os_version, '2022'), eq(parameters.os, 'linux') ) }} , or( contains(variables.CONTROL_SCENARIO, 'npm') , contains(variables.CONTROL_SCENARIO, 'all') ) )
+    timeoutInMinutes: 180 # This is for testing windows, due to the 3m between the 14 tests -> results in 42m of wasted time
```

Comment:
Noticing that that change to the k8s fork was in July fyi

If desired, we could also create another branch with a smaller timeout. I don't know why we changed to two minutes, but I feel like it's probably unnecessary at this point (Perhaps there was a latency issue in HNS that's been fixed)

Reply:
I tested with a shorter timeout from your fork and got similar results. (Roughly 20 minutes faster). I would rather keep this in and have consistent results and avoid flakes due to HNS variability.

Reply:
Does it actually take 180 mins to complete ? Can we pull npm test in parallel to all other tests (if not already)

Reply:
It takes roughly 75 - 105 minutes to complete... Main problem is that it takes 140-160 minutes to fail. If a job timeout happens before failure then we lose a considerable amount of results and debugging logs.